### PR TITLE
[ADD][10.0] account_financial_report_qweb: add analytic tags

### PR DESCRIPTION
**Pull request by jcoux**

### CODE REVIEW

**Review by p-tombez — COMMENTED** [review]
LGTM, but I have 2 questions : 
1. Can named parameters be used in the queries instead of positional ones ? Could be cleaner, easier to understand, maintain and evolve
2. I'm wondering if advanced CSS couldn't be used in the template instead of changing each value with a ```t-att-style```

**Comment by jcoux** [comment]
@p-tombez I now used named parameters in the new function/request I added.
But for others functions/requests, I prefer to not change the current working.

And for `t-att-style` attributes, that's because I compute the size of columns depending of which columns are displayed/hidden. And I don't know another way to do that.

**Review by p-tombez — APPROVED** [review]
LGTM :+1:

**Review by jackjack82 — APPROVED** [review]
Tested flow and works.

**Comment by pedrobaeza** [comment]
There are conflicts that must be solved before merging.

**Comment by jcoux** [comment]
@pedrobaeza Done

File: account_financial_report_qweb/report/general_ledger_xlsx.py
Comment (HviorForgeFlow) [inline account_financial_report_qweb/report/general_ledger_xlsx.py:54]: Update indexes for foreign_currency

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

File: account_financial_report_qweb/report/general_ledger_xlsx.py
Comment (jcoux) [reply account_financial_report_qweb/report/general_ledger_xlsx.py:54]: Fixed, and autosquash in the same time.
Thanks for check.

**Comment by leemannd** [comment]
@hveficent Could you update your review?

**Review by HviorForgeFlow — APPROVED** [review]
Merge conflict. Rebase before merge

**Comment by rafaelbn** [comment]
This is Ready to merge after a review of a PSC

**Pull request merged**
